data/resources/ui/container/creation-page.ui

```diff
@@ -265,10 +272,22 @@
 
                                               </object>
                                             </child>
-
                                           </object>
                                         </child>
+                                        <child>
```

Comment:
Command and arguments look much more connected if we move the `command_entry_row` from above here.
```suggestion
                                        <child>
                                          <object class="AdwPreferencesGroup">
                                            <child>
                                              <object class="AdwEntryRow" id="command_entry_row">
                                                <property name="activates-default">True</property>
                                                <property name="title" translatable="yes">Command</property>
                                              </object>
                                            </child>
                                          </object>
                                        </child>
                                        
                                        <child>
```
![grafik](https://user-images.githubusercontent.com/3630213/192161524-6c1a1e11-958f-434c-8040-84aadef96be2.png)

Reply:
I agree it looks much better closer together, fixed